pom.xml

```diff
@@ -239,12 +235,9 @@
                 <artifactId>jakarta.validation-api</artifactId>
                 <version>${jakarta.validation.version}</version>
             </dependency>
-            <!-- CVE-2022-25857, CVE-2022-38752 => should be removed once Quarkus uses it out of the box -->
-            <!-- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/issues/803 covers the removal of this override -->
             <dependency>
                 <groupId>org.yaml</groupId>
                 <artifactId>snakeyaml</artifactId>
-                <version>${snakeyaml.version}</version>
             </dependency>
```

Comment:
I think this dependency can be removed entirely. We don't use it directly and it will be pulled in via Jackson.

Reply:
Sure. I'll do that.